var/spack/repos/builtin/packages/mesa/package.py

```diff
@@ -66,7 +66,7 @@ class Mesa(MesonPackage):
     variant('opengles', default=False, description="Enable OpenGL ES support.")
 
     # Provides
-    provides('gl@4.5',  when='+opengl')
+    provides('gl@:4.5',  when='+opengl')
```

Comment:
Yes, otherwise paraview
5.6 does not compile with mesa.

Reply:
It can be a separate commit if you prefer.

Reply:
paraview won't or catalyst wont?  Because I do see a bug in catalyst that would cause that: `depends_on(gl@3.2')` instead of `depends_on('gl@3.2:')`.  Paraview seems to be setup correctly though.

Reply:
Yes, probably that was catalyst. Sounds good, I'll fix it that way.